extensions/wikia/Piggyback/Piggyback_body.php

```diff
@@ -4,54 +4,23 @@
 use Wikia\Service\User\Auth\CookieHelper;
```

Comment:
These imports/aliases are no longer needed as the code using them has been removed. I have checked it in the previous revision of this file, but a second pair of eyes will not hurt.